src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -241,18 +241,19 @@ void set(Event e, String message) {
 				((UnknownCommandEvent) e).setMessage(message);
 			}
 		};
-		@SuppressWarnings("null")
-		static ArrayList<String> patterns;
-		@SuppressWarnings("null")
+		
+		static ArrayList<String> patterns = new ArrayList<>();
 		static EnumSet<MessageType> supportedTypes;
 		
 		static {
 			MessageType[] types = values();
-			for (int i = 0; i < types.length; i++) {
+			supportedTypes = EnumSet.allOf(MessageType.class);
```

Comment:
sorry my example was wrong but it would be better to use `noneOf` and then `add` instead of `allOf` and `remove`

Reply:
Okay